src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -2889,17 +2889,17 @@ namespace GraphQL.Validation
     {
         public static readonly System.Collections.Generic.IEnumerable<GraphQL.Validation.IValidationRule> CoreRules;
         public DocumentValidator() { }
-        [return: System.Runtime.CompilerServices.TupleElementNames(new string?[]?[] {
+        [return: System.Runtime.CompilerServices.TupleElementNames(new string[] {
```

Comment:
🤔 Looks like a bug in NRT subsystem of ApiApprover.

Reply:
I have some NRT analysis code in the GraphQL.DI project we can copy if need be, with 100% code coverage and about "a million" tests for different NRT rules.  You can "enumerate" any type to walk the NRT annotations within the type.

Reply:
I have some NRT analysis code in the GraphQL.DI project we can copy if need be, with 100% code coverage and about "a million" tests for different NRT rules. You can "enumerate" any type to walk the NRT annotations within the type.